crates/interfaces/src/provider.rs

```diff
@@ -26,19 +27,16 @@ pub enum ProviderError {
         storage_key: B256,
     },
     /// The block number was found for the given address, but the changeset was not found.
-    #[error("account {address:?} ChangeSet for block #{block_number} does not exist")]
+    #[error("account {address} ChangeSet for block #{block_number} does not exist")]
```

Comment:
it still outputs the full address and formatted with ellipsis only with `#` modifier, right?

Reply:
Yes, Display uses checksum too: https://docs.rs/alloy-primitives/latest/alloy_primitives/struct.Address.html